BDArmory/Bullets/PooledBullet.cs

```diff
@@ -203,6 +202,14 @@ void OnWillRenderObject()
 
         void FixedUpdate()
         {
+            //floating origin and velocity offloading corrections
+            if (!FloatingOrigin.Offset.IsZero() || !Krakensbane.GetFrameVelocity().IsZero())
```

Comment:
The OffsetNonKrakensbane variable is not reset if no correction is performed, so checking via different variables.